### PR TITLE
fix LICENSE and NOTICE

### PR DESCRIPTION
**Pull request by mechanomi**
another PR for this

does exactly the same as in crate/crate-admin/pull/553

### CODE REVIEW

File: NOTICE [hunk 1]
```diff
@@ -1,100 +1,13 @@
-Crate Data Admin Interface
+Crate.io Sphinx Theme
+Copyright 2013-2018 CRATE Technology GmbH ("Crate")
 
-Copyright 2013-2015 Crate.IO GmbH
-Prinzessinnenstr. 19-20, 10969 Berlin, Germany
-(referred to in this notice as "Crate")
 
+Licensed to CRATE Technology GmbH (referred to in this notice as "Crate")
```
Comment (chaudum) [inline NOTICE]: Crate.io Inc?

Comment (jodok) [reply NOTICE]: i'm working with naomi on this. IP is in the Crate.IO GmbH

**Comment by nslater** [comment]
leaving a note here to say this pr is blocked on the trello cards

**Comment by mechanomi** [comment]
closing due to inactivity. adding link to trello cards

**Comment by mechanomi** [comment]
put this back on the impeds board

**Pull request merged**